src/Compilers/CSharp/Test/Semantic/SourceGeneration/StateTableTests.cs

```diff
@@ -89,11 +90,18 @@ public void Node_Builder_Can_Add_Entries_From_Previous_Table()
 
             var expected = ImmutableArray.Create((10, EntryState.Added, 0), (11, EntryState.Added, 1), (2, EntryState.Cached, 0), (3, EntryState.Cached, 1), (20, EntryState.Modified, 0), (21, EntryState.Modified, 1), (22, EntryState.Modified, 2), (6, EntryState.Removed, 0));
             AssertTableEntries(newTable, expected);
-            Assert.Equal(new[] { 2, 3 }, cachedEntries.ToImmutableArray());
-            Assert.Equal(6, Assert.Single(removedEntries));
+            Assert.Equal(new[] { 2, 3 }, YieldItems(cachedEntries.Items));
```

Comment:
.Items is now returning a OneOrMany, so this just translates it so that the test can easily see all the values in that.